utils/gcp.py

```diff
@@ -477,6 +484,95 @@ def bigquery_xcom_query(
     )
 
 
+def bigquery_table_sensor(
+    task_id: str,
+    table_id: str,
+    gcp_conn_id: str = "google_cloud_airflow_gke",
```

Comment:
I'm not sure if using this `google_cloud_airflow_gke` connection is appropriate, since the sensors will run directly in Airflow and not in GKE pods.

Is there a different connection that would be better to use? (we'd want it to have access to all the same BigQuery projects/datasets/tables as the `google_cloud_airflow_gke` connection)

Reply:
I believe `google_cloud_shared_prod` should be used for this. You are right that `google_cloud_airflow_gke` will not work as it is used only to access the GKE cluster.